lib/DW1000_library/src/DW1000.cpp

```diff
@@ -109,7 +109,7 @@ const byte DW1000Class::BIAS_900_64[] = {147, 133, 117, 99, 75, 50, 29, 0, 24, 4
 	const SPISettings DW1000Class::_fastSPI = SPISettings(16000000L, MSBFIRST, SPI_MODE0);
 #endif
 const SPISettings DW1000Class::_slowSPI = SPISettings(2000000L, MSBFIRST, SPI_MODE0);
-const SPISettings* DW1000Class::_currentSPI = &_fastSPI;
+const SPISettings* DW1000Class::_currentSPI = &_slowSPI;
```

Comment:
shouldn't we leave it as `_fastSPI`? It only matters for us when using the logic analyzer. Especially for ESP32 it's good to have it as fast as possible, since the SPI stays on the PCB - there shouldn't be any issues with higher speeds.

Reply:
This will be overwritten during init :sweat_smile: so it does not matter what we write here, but for clarity we can leave it for _fastSPI

Reply:
ah, yeah, I forgot but let's revert it then since this change is just confusing (like their entire setup :upside_down_face: )

Reply:
Yes